airbyte-integrations/connectors/source-shopify/source_shopify/spec.json

```diff
@@ -98,12 +98,27 @@
         "description": "Defines which API type (REST/BULK) to use to fetch `Transactions` data. If you are a `Shopify Plus` user, leave the default value to speed up the fetch.",
         "default": false
       },
+      "job_product_variants_include_pres_prices": {
+        "type": "boolean",
+        "title": "Add `Presentment prices` to Product Variants",
+        "description": "If enabled, the `Product Variants` stream attempts to include `Presentment prices` field (may affect the performance).",
+        "default": true
+      },
       "job_termination_threshold": {
         "type": "integer",
         "title": "BULK Job termination threshold",
         "description": "The max time in seconds, after which the single BULK Job should be `CANCELED` and retried. The bigger the value the longer the BULK Job is allowed to run.",
-        "default": 3600,
-        "minimum": 1
+        "default": 7200,
+        "minimum": 3600,
```

Comment:
Curious, is there a specific reason the minimum needs to be 3600 or greater than 1? Adding this requirement means large jobs significantly increase the time to which they arrive at a small enough slice that can run without failure.

Reply:
@jblakeman Thank you for raising this; we selected the values from what we got on our side and are happy to adjust them now that we have your feedback. I'll add this to the next source-shopify update.